python/example_code/medical-imaging/README.md

```diff
@@ -39,10 +39,24 @@ python -m pip install -r requirements.txt
 
 Code excerpts that show you how to call individual service functions.
 
+* [Add a tag to a resource](medical_imaging_basics.py#L388) (`TagResource`)
+* [Copy an image set](medical_imaging_basics.py#L334) (`CopyImageSet`)
 * [Create a data store](medical_imaging_basics.py#L22) (`CreateDatastore`)
-* [Delete a data store](medical_imaging_basics.py#L78) (`DeleteDatastore`)
-* [Get data store properties](medical_imaging_basics.py#L41) (`GetDatastore`)
-* [List data stores](medical_imaging_basics.py#L60) (`ListDatastores`)
+* [Delete a data store](medical_imaging_basics.py#L85) (`DeleteDatastore`)
+* [Delete an image set](medical_imaging_basics.py#L366) (`DeleteImageSet`)
+* [Get an image frame](medical_imaging_basics.py#L258) (`GetImageFrame`)
+* [Get data store properties](medical_imaging_basics.py#L42) (`GetDatastore`)
+* [Get image set properties](medical_imaging_basics.py#L203) (`GetImageSet`)
+* [Get metadata for an image set](medical_imaging_basics.py#L226) (`GetImageSetMetadata`)
+* [Get the import job properties](medical_imaging_basics.py#L132) (`GetDICOMImportJob`)
+* [List data stores](medical_imaging_basics.py#L62) (`ListDatastores`)
+* [List image set versions](medical_imaging_basics.py#L285) (`ListImageSetVersions`)
+* [List import jobs for a data store](medical_imaging_basics.py#L153) (`ListDICOMImportJobs`)
+* [Lists all tags for a resource](medical_imaging_basics.py#L424) (`ListTagsForResource`)
+* [Remove a tag from a resource](medical_imaging_basics.py#L406) (`UntagResource`)
+* [Search image sets](medical_imaging_basics.py#L177) (`SearchImageSets`)
+* [Start importing bulk data into a data store](medical_imaging_basics.py#L102) (`StartDICOMImportJob`)
+* [Update image set metadata](medical_imaging_basics.py#L307) (`UpdateImageSetMetadata`)
 
 ## Run the examples
 
```

Comment:
My only comment here is that with all of these actions, shouldn't there be a main method and instructions on how to run them in order to match the rest of our examples? My suggestion is to add something to let the customer execute the actions, but that can be in a subsequent PR.